### PR TITLE
#159987397 APIs documentation using swagger

### PR DESCRIPTION
**Pull request by DanielAdek**
#### What does this PR do?
Setup APIs documentation using swagger `https://metis-ah-staging-pr-7.herokuapp.com/api-docs`
#### Description of Task to be completed?
* Install swagger-ui-express
* Install cors
* create a swagger.json file in the root directory of your project
* on your browser open editor.swagger.io
* click on convert and save as Json button, in file navigation bar of the site above
* after it has downloaded, copy the code in the file, into the swagger.json file in your project
* edit the code according to your project.
#### How should this be manually tested?
1. Clone repository
2. On your terminal run this command git checkout ft-swagger-docs-159987397
3. run npm install to install packages
4. run npm start
5. open your browser and enter `localhost:8000/api-docs`
#### Any background context you want to provide?
This setup is a documentation of all the APIs endpoint you documented in the swagger.json file
#### What are the relevant pivotal tracker stories?
#159987397

### CODE REVIEW

**Review by jojitoon — APPROVED** [review]
good work. swagger docs running.

File: swagger.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      }
+    },
+    "securityDefinitions": {
+      "petstore_auth": {
+        "type": "oauth2",
+        "authorizationUrl": "http://petstore.swagger.io/oauth/dialog",
+        "flow": "implicit",
+        "scopes": {
+          "write:pets": "modify pets in your account",
+          "read:pets": "read your pets"
```
Comment (iAmao) [inline swagger.json]: Remove this config

File: swagger.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    ],
+    "schemes": [
+      "https",
+      "http"
+    ],
+    "paths": {
+      "/": {
+        "get": {
+          "tag": [
+            "sims"
```
Comment (iAmao) [inline swagger.json]: remove sims tag

File: swagger.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "description": "Auto's Heven [https://metis-ah-staging.herokuapp.com/](https://metis-ah-staging.herokuapp.com/) is a platform that allows you to share your knowledge across the world",
```
Comment (iAmao) [inline swagger.json]: Fix typo "Auto's Heven" should be Author's Haven

File: .eslintrc.json [hunk 4]
```diff
@@ -14,7 +14,7 @@
     "no-param-reassign": 0,
     "comma-dangle": 0,
     "curly": ["error", "multi-line"],
-    "import/no-unresolved": [2, { "commonjs": true }],
+    "import/no-unresolved": [2, { "commonjs": true}],
```
Comment (iAmao) [inline .eslintrc.json]: Fix blank space before } to follow airbnb style guides

**Review by iAmao — CHANGES_REQUESTED** [review]

**Review by iAmao — APPROVED** [review]

**Pull request merged**